Vector.java

```diff
@@ -0,0 +1,41 @@
+import java.io.*;
+import java.util.*;
+import java.lang.Math; 
+
+public class Vector {
+	
+	List<Double> numbers;
+		
+	Vector (List<Double> numbers) {
+			
+		numbers = numbers;
+			
+	}
+		
+	public double distance(Vector vector2) { // returns distance between two vectors
```

Comment:
I think this should be a static method, rather than an instance method.  The signature is cleaner if it's
```
public static double distance(Vector vector1, Vector vector2) ...
```